data-collection-consent/consent/consent-lib.js

```diff
@@ -0,0 +1,287 @@
+/* exported consent */
+// eslint-disable-next-line no-unused-vars
+var consent = (function() {
+  // This library is wrapped in an immediately invoked function expression
+  // (IIFE) in order to prevent it's utility functions from polluting the global
+  // namespace.
+
+  /**
+   * Minimal shim to make the `browser` global work as expected in Chrome.
+   */
+  const browser = globalThis.browser ?? chrome;
+
+  /**
+   * A "no operation" function. It doesn't do anything.
```

Comment:
Not sure if this is a rhetorical question. Just in case, noop functions are useful in situations where a function is called unconditionally, but you don't want that operation to have any side effects (see [NOP (code)](https://en.wikipedia.org/wiki/NOP_(code))).

In this case I think we're probably better off just dropping it entirely. I'll do that in a follow-up commit

Reply:
@dotproto, not entirely. We promote the extension examples as part of our get-started flow. Therefore, it's entirely possible that someone who visits here is not particularly skilled in JavaScript development and so may be bemused as to why you have something that does nothing.